reporter/html.js

```diff
@@ -108,6 +108,15 @@ function id( name ) {
 	return document.getElementById && document.getElementById( name );
 }
 
+function stopTests() {
+  var stopButton = id( "qunit-stop-tests-button" );
+  if ( stopButton ) {
+    stopButton.innerHTML = "Stopping";
```

Comment:
It says `Stop` when the button is created and added to the toolbar, and then set again to `Stop` when the tests are done.